### ISSUE TITLE
Getting a executing error

### ISSUE THREAD

**Issue by Cedirna** [PoE-TradeMacro/POE-TradeMacro#888]
Can someone help me about this error ? Thats happened me after i try to update

![errr](https://user-images.githubusercontent.com/46615767/51068344-cf5f9d00-162d-11e9-8fe5-4f5c3e46e457.png)

**Comment by Cedirna** [comment]
Solved it, i changed directory from desktop to c:/program files.

**Comment by DarkLord1610** [comment]
I had the same problem and I got it solved.
first, it started with failing to use POETM in-game, then after re-launching it and updating, I had a cookie read error, following that I re-installed the macro to find the same error window you posted followed by a crush.
How To Solve:
I downloaded all fresh files, POETM and AHK
I moved my Trade Macro folder from my desktop (where I used to have it with the past releases) into one of my partitions (D:/PoE Trade Macro/POE-TradeMacro-2.12.4)
I re-installed AHK (just in case), then restarted my PC.
it works fine now.
try it.
